rfc/src/rfcs/0001-mir-linker.md

```diff
@@ -97,7 +97,7 @@ The two options to be included in this RFC is starting from all local harnesses
 The `kani-compiler` behavior will be customizable via a new flag:
 
   ```
-  --reachability=[ harnesses | pub_fns |  none | legacy ]
+  --reachability=[ harnesses | pub_fns |  none | legacy | tests ]
```

Comment:
Should we remove `legacy`?

Reply:
We still have `legacy` which is used internally to test code generation of the standard library.

Reply:
OK.